lib/ts/Diags.h

```diff
@@ -226,6 +242,10 @@ class Diags
   mutable ink_mutex tag_table_lock; // prevents reconfig/read races
   DFA *activated_tags[2];           // 1 table for debug, 1 for action
 
+  // Scrubbing variables -- replaces predefined regexp matches with the replacement string
+  bool scrub_enabled = false;
+  Scrubber *scrubber = nullptr;
```

Comment:
As discussed, due to namespace issues in `Vec.h`, using `unique_ptr` is more trouble than it's worth.